OoTAdjuster.py

```diff
@@ -231,6 +233,8 @@ def adjust(args):
     rom.write_to_file(decomp_path)
     compress_rom_file(decomp_path, comp_path)
     os.remove(decomp_path)
+    if delete_zootdec:
```

Comment:
I think this would be safer to wrap in a "finally" or "with" block, to prevent future issues if an exception is raised in this code block.